lib/compat/wordpress-6.1/blocks.php

```diff
@@ -90,7 +90,7 @@ function gutenberg_block_type_metadata_view_script( $settings, $metadata ) {
 		$view_script_version      = isset( $view_asset['version'] ) ? $view_asset['version'] : false;
 		$result                   = wp_register_script(
 			$view_script_handle,
-			gutenberg_url( str_replace( gutenberg_dir_path(), '', $view_script_path ) ),
+			gutenberg_url( str_replace( wp_normalize_path( gutenberg_dir_path() ), '', $view_script_path ) ),
```

Comment:
Finally, normalize `gutenberg_dir_path()` so that the forward-slashed `$view_script_path` can be replaced correctly.